pyproject.toml

```
@@ -1,7 +1,48 @@
 [build-system]
-requires = ["setuptools>=45", "setuptools_scm[toml]>=6.2"]
+requires = ["setuptools>=64", "setuptools_scm[toml]>=6.2"]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "pygmt"
+description = "A Python interface for the Generic Mapping Tools"
+readme = "README.rst"
+requires-python = ">=3.8"
+license = {text = "BSD License"}
+authors = [{name = "The PyGMT Developers", email = "pygmt.team@gmail.com"}]
+maintainers = [{name = "The PyGMT Developers", email = "pygmt.team@gmail.com"}]
+keywords = []
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Intended Audience :: Science/Research",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Education",
+    "Topic :: Scientific/Engineering",
+    "Topic :: Software Development :: Libraries",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "License :: OSI Approved :: BSD License",
+]
+dependencies = ["numpy>=1.20", "pandas", "xarray", "netCDF4", "packaging"]
```

Small suggestion to have this as multi-line to have a cleaner diff when we bump the minimum NumPy version.

```suggestion
dependencies = [
    "numpy>=1.20",
    "pandas",
    "xarray",
    "netCDF4",
    "packaging"
]
```